README.md

```diff
@@ -49,7 +49,23 @@ $ sudo GOPATH=$GOPATH CNI_PATH=$GOPATH/bin ./kube-spawn setup --image=coreos --n
 $ sudo GOPATH=$GOPATH CNI_PATH=$GOPATH/bin ./kube-spawn init
 ```
 
-## Running local Kubernetes builds
+Now that you're up and running, you can start using it.
+
+## Deploying to your local cluster
+
+`kube-spawn` creates `tmp/` inside the directory you run it in.
```

Comment:
Is we are now creating the `.kube-spawn` directory, why don't we just put that there too?